### ISSUE TITLE
Consider whether we want to collect reporter's details on the data breach form

### ISSUE THREAD

**Issue by WilliamWDTK** [mysociety/whatdotheyknow-theme#1777]
On occasions, reports may be submitted through the data breach form with no contact details. We currently only ask for the email address of the Data Protection Officer, and we don't require this field to be filled.

In some instances, particularly if the report is being not being made on behalf of the authority concerned, we may want to contact the person reporting the issue, rather than the authority, especially if not enough information is provided. We can currently only do this is the user is signed in, or independently chooses to provide their contact details.

This is not _currently_ a development ticket, but a ticket to assess whether we want to make the change.

**Comment by mdeuk** [comment]
**TL;DR:** I think it'd be a useful improvement to have the form collect contact details In the usual way, and continue to _optionally_ ask for DPO details. 

I think it would certainly be beneficial to consistently collect details of _who_ is filling the form in, as there is currently some ambiguity - e.g. unless otherwise communicated in the free-text, do we know it is _definitely_ the DPO who is writing?

In many cases, it might be someone else, or even a third party, so, IMHO, having the distinction would help with the "what next" [^1].

[^1]: and perhaps also ensure that the first contact with a public body is "thanks for your report", when they may be none-the-wiser.

**Comment by WilliamWDTK** [comment]
Just to clarify my own view on this, I think it would be a helpful improvement to collect the form-filler's details, as long as we make it explicit that's what we're asking for, and what we want in the DPO field.

**Comment by WilliamWDTK** [comment]
I really think we should collect the details of the reporter. I wouldn't say there is much reason not to, and we've had several cases where it would have been useful to know who filled it out, either to contact them, or to just have an idea of who might have reported it.
